src/main.rs

```diff
@@ -485,6 +506,28 @@ impl AsyncComponent for App {
             gtk::glib::Propagation::Stop
         });
 
+        #[cfg(target_os = "linux")]
+        let tray_img = Icon::from_png_bytes(&TRAY_IMAGE.to_vec()).expect("icon");
+        #[cfg(target_os = "windows")]
+        let tray_img = Icon::from_resource(1, None).expect("icon");
+
+        let sender_c = sender.clone();
+        let tray = TrayIconBuilder::new()
+            .with_icon(tray_img)
+            .with_tooltip("Demo System Tray")
+            .with_menu(build_menu())
+            .build(move |tray_event| match tray_event {
+                TrayEvent::Menu(signal) => match signal {
+                    TrayMenuSignal::Open => sender_c.input(AppInput::ShouWindow),
+                    TrayMenuSignal::Quit => sender_c.input(AppInput::Quit),
+                },
+                TrayEvent::Tray(e) => match e {
+                    ClickType::Double => sender_c.input(AppInput::ShouWindow),
+                    _ => {}
+                },
+            })
+            .expect("build tray icon");
```

Comment:
You can avoid custom names like `sender_c` like this:
```suggestion
        let tray = TrayIconBuilder::new()
            .with_icon(tray_img)
            .with_tooltip("Demo System Tray")
            .with_menu(build_menu())
            .build({
                let sender = sender.clone();
            
                move |tray_event| match tray_event {
                    TrayEvent::Menu(signal) => match signal {
                        TrayMenuSignal::Open => sender.input(AppInput::ShouWindow),
                        TrayMenuSignal::Quit => sender.input(AppInput::Quit),
                    },
                    TrayEvent::Tray(e) => match e {
                        ClickType::Double => sender.input(AppInput::ShouWindow),
                        _ => {}
                    },
                }
            })
            .expect("build tray icon");
```

Reply:
done, and change double to left click,   better compatible